src/Integration/UnintrusiveBinding/UnintrusiveConfigurationProvider.cs

```diff
@@ -23,13 +23,9 @@
 
 namespace SonarLint.VisualStudio.Integration.NewConnectedMode
 {
-    internal interface IConfigurationProviderService : IConfigurationProvider, ILocalService
```

Comment:
This is the original of all of the rest of the changes: I've dropped the `IConfigurationProviderService` marker interface.

That meant I had to:
1) update the `VsSessionHost` so that it doesn't return it as a local service any more, and
2) changed the remaining few classes that still fetched it as a service so they now get it via MEF.